src/test/kotlin/step2/StringCalculatorTest.kt

```diff
@@ -0,0 +1,65 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringCalculatorTest {
+    private val calculator = StringCalculator()
+
+    @ParameterizedTest
+    @ValueSource(strings = ["", "  "])
+    fun `입력값이 null이거나 빈 공백 문자일 경우 IllegalArgumentException이 발생한다`(input: String) {
+        assertThatThrownBy { calculator.execute(input) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["3 % 4", "5 , 3"])
+    fun `사칙 연산 기호가 아닌 경우 IllegalArgumentException이 발생한다`(input: String) {
+        assertThatThrownBy { calculator.execute(input) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
+
+    @Test
+    fun `덧셈이 가능하다`() {
+        val input = "5 + 3"
+        val actual = calculator.execute(input)
+        val expected = 8.0
+        assertThat(actual).isEqualTo(expected)
+    }
+
+    @Test
+    fun `뺄셈이 가능하다`() {
+        val input = "5 - 3"
+        val expected = 2.0
+        val actual = calculator.execute(input)
+        assertThat(actual).isEqualTo(expected)
```

Comment:
저는 개인적으로 given/when/then 표기법을 사용하곤합니다
준비/실행/검증 부를 나눠놓으면 좀더 가독성 좋을거 같아요!
```suggestion
        // given
        val input = "5 - 3"
        val expected = 2.0
        // when 
        val actual = calculator.execute(input)
        // then
        assertThat(actual).isEqualTo(expected)
```